core/src/main/java/hudson/Functions.java

```diff
@@ -662,18 +662,13 @@ public static String validateIconSize(String iconSize) throws SecurityException
     }
 
     /**
-     * Gets the suffix to use for YUI JavaScript.
+     * @deprecated yui has been removed, do not call this
      */
+    @Deprecated(forRemoval = true, since = "TODO")
     public static String getYuiSuffix() {
-        return DEBUG_YUI ? "debug" : "min";
+        return "min";
     }
 
-    /**
-     * Set to true if you need to use the debug version of YUI.
-     */
-    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "for script console")
-    public static boolean DEBUG_YUI = SystemProperties.getBoolean("debug.YUI");
```

Comment:
Is this passing PCT on plugins with a test harness that predates https://github.com/jenkinsci/jenkins-test-harness/pull/874?

Reply:
I'll check once the build passes, (flaked before).

Will see impact and decide.

Reply:
👉 https://github.com/jenkinsci/bom/pull/4176

Reply:
https://ci.jenkins.io/job/Tools/job/bom/job/PR-4176/2/testReport/

Quite a lot failed, scrolling through here's a selection:

active-directory, pipeline-build-step,JiraTestResultReporter,analysis-model-api,oauth-credentials,database-sqlite,ant,

I've restored it